lib/desired-caps.js

```diff
@@ -0,0 +1,7 @@
+let geckodriverConstraints = {
+  geckodriverPort: {
```

Comment:
I still think there should be a possibility to provide full set of driver options in capabilities

Reply:
What do you mean?

Reply:
I mean we need capability like `options`, where we could manually provide geckdriver capabilities as object. These capabilities will then be used as defaults unless overridden by other explicit caps, for example the `noReset` one